locale/ru/LC_MESSAGES/clipboard-history@alexsaveau.dev.po

```diff
@@ -145,33 +141,4 @@ msgstr "Оба"
 
 #: prefs.js:315
 msgid "Neither"
-msgstr ""
-
-#, fuzzy
-#~ msgid "Previous page"
-#~ msgstr "Предыдущая запись"
-
-#, fuzzy
-#~ msgid "Next page"
-#~ msgstr "Следующая запись"
-
-#~ msgid "Settings"
-#~ msgstr "Настройки"
-
-#~ msgid "Text will be here"
-#~ msgstr "Текст будет здесь"
-
-#~ msgid "History Size"
-#~ msgstr "Размер истории"
-
-#~ msgid "Refresh Interval (ms)"
-#~ msgstr "Интервал обновления (мс)"
-
-#~ msgid "Max cache file size (kb)"
-#~ msgstr "Максимальный размер кэша (кб)"
-
-#~ msgid "Cache only favorites"
-#~ msgstr "Кэшировать только избранное"
-
-#~ msgid "Move item to the top after selection"
-#~ msgstr "Переместить запись в начало после выделения"
```

Comment:
Can you add these back? If we ever use one of those strings again we won't need someone to translate them again.

Reply:
Done

Reply:
Thanks!